content/desktop/contributing-and-collaborating-using-github-desktop/stashing-changes.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Stashing changes
+intro: You can temporarily save your changes without committing the changes to a branch by stashing the changes.
+versions:
+  free-pro-team: '*'
+---
+
+### About stashed changes
+
+To apply your changes to your repository, you must save the files and then commit the changes to a branch. If you have saved changes that you are not ready to commit yet, you can stash the changes for later. When you stash changes, the changes are temporarily removed from the files and you can choose to restore or discard the changes later. You can only stash one set of changes at a time with {% data variables.product.prodname_desktop %}. If you use {% data variables.product.prodname_desktop %} to stash changes, all unsaved changes will be stashed. After you stash changes on a branch, you can safely change branches or make other changes to your current branch.
+
+You can also stash changes when you switch branches. For more information, see "[Managing branches](/desktop/contributing-to-projects/managing-branches#switching-between-branches)."
```

Comment:
"You can also stash changes when you switch branches" - I think this reads a little oddly because this falls within what you've described in the first paragraph, rather than being _another_ situation where you can stash. Maybe change this to something like:

```suggestion
A common situation where you might choose to stash your changes is when you want to switch branches, but you don't want to commit your changes yet. For more information about switching branches, see "[Managing branches](/desktop/contributing-to-projects/managing-branches#switching-between-branches)."
```

Reply:
I think what's unstated now is that if you have uncommitted changes and use Desktop to switch branches, the app will prompt you to save changes with a slightly different flow, which is described in the link. I'll make that a bit more clear and why it's being brought up in this article 👍